go.mod

```diff
@@ -157,3 +156,5 @@ require (
 )
 
 replace github.com/grpc-ecosystem/grpc-gateway => github.com/temporalio/grpc-gateway v1.17.0
+
+replace go.temporal.io/sdk => github.com/dandavison/temporalio-sdk-go v1.30.0
```

Comment:
Does this PR need to be updated for #420 and have this removed?

Reply:
I'll close this one and open a PR adding the release candidate cherry pick to main.